modules/azure-gaming/game-dev-vmss/nestedtemplates/virtualNetworks.bicep

```diff
@@ -0,0 +1,33 @@
+param location                 string
```

Comment:
nit, but why is the formatting this style? This is different that how the bicep formatter will organize things. I'd rather us be consistent with the formatter

Reply:
I can revert it.

I have a column alignment setting in my auto formatter... I find it much easier to read blocks like this, and have always used it when formatting JSON or YAML.

Reply:
<img width="262" alt="image" src="https://user-images.githubusercontent.com/9027725/182717940-dc42fd53-966e-48ff-a0f1-b7a6392128b5.png">

Reply:
compared to 
<img width="281" alt="image" src="https://user-images.githubusercontent.com/9027725/182718038-f4abcd8c-5f2a-4012-a6c7-4bf07e767af0.png">

Reply:
I might have done to much SAS coding in school.....